elastica/rigidbody/rigid_body.py

```diff
@@ -104,3 +104,76 @@ def compute_rotational_energy(self):
             "ijk,jk->ik", self.mass_second_moment_of_inertia, self.omega_collection
         )
         return 0.5 * np.einsum("ik,ik->k", self.omega_collection, J_omega).sum()
+
+    def compute_position_of_point(self, point, index: int = 0):
```

Comment:
Let's remove `compute_position_of_point` and  `compute_velocity_of_point` from RigidBody and include them in joint file as functions.

Reply:
Implemented in https://github.com/GazzolaLab/PyElastica/pull/149/commits/74a9c9d3f1511d9aed73a7f7f7b804cf0bebb284